test/integration/short/metadata-tests.js

```diff
@@ -165,7 +167,7 @@ describe('Metadata', function () {
             assert.ok(udt);
             assert.strictEqual(udt.fields.length, 2);
             assert.strictEqual(udt.fields[1].name, 'name');
-            assert.strictEqual(udt.fields[1].type.code, types.dataTypes.varchar);
+            assert.ok(udt.fields[1].type.code === types.dataTypes.varchar || udt.fields[1].type.code === types.dataTypes.text);
```

Comment:
Previously `varchar` and `text` were returned as Fqcn `UTF8Type`, now its either `text` or `varchar` depending on how it was defined.